### PR TITLE
typos and grammar in intro

### PR DESCRIPTION
**Pull request by ethers**

### CODE REVIEW

File: source/introduction/what-is-ethereum.rst [hunk 1]
```diff
@@ -3,9 +3,9 @@ A next generation blockchain
...
 
-A few years into the bitoin experiment novel projects started to show interest in using the bitcoin network for purposes other than financial transfer. This provided the necessary inspiration Ethereum's inventor, Vitalik Buterin needed to realise that technial requirements for all these ventures could be subsumed under a high level system only if the right abstraction is found.
+A few years into the bitcoin experiment novel projects started to show interest in using the bitcoin network for purposes other than financial transfer. This provided the necessary inspiration Ethereum's inventor, Vitalik Buterin, needed to realise that the technical requirements for all these ventures could be subsumed under a high level system, only if the right abstraction is found.
```
Comment (hughlang) [inline source/introduction/what-is-ethereum.rst]: Needs comma and some clarification.
"A few years into the bitcoin experiment, developers began to show interest"

It is currently a bit wordy. You can substitute "developers" for whatever term you think is right. Possibly

File: source/introduction/what-is-ethereum.rst [hunk 2]
```diff
@@ -3,9 +3,9 @@ A next generation blockchain
...
...kchain technology is the technological basis of Bitcoin, first described as a coherent cryptoeconomic system by its mysterious author Satoshi Nakamoto in his white paper "Bitcoin: A Peer-to-Peer Electronic Cash System", published in 2009. While the use of blockchains for more general uses is already discussed in the original paper, it was not until a few years later that blockchain technology emerged as a generic term to designate the cryptoeconomic system composed of a transparent open ledger or database representing a state of a complex system, updated via sending transactions to the guardians of the blockchain. The guardians are nodes of a decentralised peer to peer network. In Bitcoin's case the open ledger is simply conceived of as a table of account balances, and transactions are value transfers. Due to the decentralised nature of this peer to peer system, secured by cryptography and driven by algorithmically enforced economic incentives coupled with scarcity and controlled issuance (predictable inflation) coded in the protocol, bitcoin is suitable to serve as sound digital money without central bank or other power behind it. This allows trustless finance between individuals.
```
Comment (hughlang) [inline source/introduction/what-is-ethereum.rst]: All of this is great information, yet exceedingly complex as an intro.
 
I wrote the first draft of this page and I had assumed that it should be readable by a newcomer while still being technically accurate for those who have already studied Bitcoin and blockchain.

 Just my thoughts.

Comment (Souptacular) [inline source/introduction/what-is-ethereum.rst]: I agree with this sentiment. the guide should be targeted to brevity and simplicity over technical exactness and too many details.

Comment (Souptacular) [inline source/introduction/what-is-ethereum.rst]: Just noticed this PR was just for typos. That may have been already fixed in a different PR, checking now...

**Comment by Souptacular** [comment]
Changes were fixed in another PR. Closing this one. Thanks @ethers ! Sorry I didn't merge this in time!
